File: Apps/DriveSwitch/app/src/main/java/com/driveswitch/Address.java
package com.driveswitch;

import org.json.JSONException;
import org.json.JSONObject;

public class Address
{
    public Address()
    {
        CountryId = Constants.EmptyObjectId;
        StateId = Constants.EmptyObjectId;
        CountyId = Constants.EmptyObjectId;
        CityId = Constants.EmptyObjectId;
        ZipCode = "00000";
        TimeZoneId = Constants.EmptyObjectId;
        Address1 = Constants.EmptyObjectId;
        Address2 = Constants.EmptyObjectId;
    }

    public Address(JSONObject addressObj)
    {
        try
        {
            CountryId = addressObj.getString("CountryId");
            StateId = addressObj.getString("StateId");
            CountyId = addressObj.getString("CountyId");
            CityId = addressObj.getString("CityId");
            ZipCode = addressObj.getString("ZipCode");
            TimeZoneId = addressObj.getString("TimeZoneId");
            Address1 = addressObj.getString("Address1");
            Address2 = addressObj.getString("Address2");
        }
        catch (JSONException e)
        {
            e.printStackTrace();
        }
    }

    public String CountryId;
    public String StateId;
    public String CountyId;
    public String CityId;
    public String ZipCode;
    public String TimeZoneId;
    public String Address1;
    public String Address2;
}

File: Libraries/Data.Repositories.MongoDB/Models/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using MongoDB;
using MongoDB.Bson;
using MongoDB.Driver;

namespace Data.Repositories.Models
{
    public class Address
    {
        public Address()
        {
            CountryId = Constants.DefaultCountryId;
            StateId = ObjectId.Empty;
            CountyId = ObjectId.Empty;
            CityId = ObjectId.Empty;
            ZipCode = 00000;
            TimeZoneId = ObjectId.Empty;
            Address1 = "";
            Address2 = "";
        }
        public ObjectId CountryId { get; set; }
        public ObjectId StateId { get; set; }
        public ObjectId CountyId { get; set; }
        public ObjectId CityId { get; set; }
        public Int32 ZipCode { get; set; }
        public ObjectId TimeZoneId { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
    }
}